examples/docs/src/main.rs

```diff
@@ -143,6 +143,7 @@ fn main() {
     router.add("/basics/templating", templating_page);
     router.add("/basics/mixins", mixins_page);
     router.add("/basics/events", event_handling_page);
+    router.add("/basics/browser-instance", browser_instance_page);
```

Comment:
Not sure we need this page though, Cant we just add a the info here to the home page?

Reply:
Yea sure, I just thought we could do something like yew did where they include stuffs like web_sys and wasm_bindgen in separate pages.